<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RconsoleWidget</class>
 <widget class="QDockWidget" name="RconsoleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>R Script Editor</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0" colspan="9">
     <widget class="QTextEdit" name="editor"/>
    </item>
    <item row="9" column="0" colspan="9">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="outputTab">
       <attribute name="title">
        <string>Output</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTextEdit" name="outputLog"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="errorTab">
       <attribute name="title">
        <string>Error Log</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPlainTextEdit" name="errorLog"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="btnRun">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="btnOpen">
      <property name="text">
       <string>Load Script</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btnSave">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Save Script</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="7">
     <widget class="QLabel" name="fileLabel">
      <property name="text">
       <string>File:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
